resources/exec.go

```diff
@@ -392,6 +393,46 @@ func (obj *ExecRes) AutoEdges() (AutoEdge, error) {
 			},
 			path: x, // what matters
 		})
+		data = append(data, &FileUID{
+			BaseUID: BaseUID{
+				Name:     obj.GetName(),
+				Kind:     obj.GetKind(),
+				Reversed: &reversed,
+			},
+			path: x, // TODO: check that we handle relative paths in FileRes
```

Comment:
This is a good question :P 
Would you like to open an issue ticket about this? We should probably check for this in Validate() since AFAIK relative paths don't make sense in file resources. Am I right ? Wrong? Let's discuss more in the issue.

Reply:
Opened #271 in that regard :)